types/react-native/index.d.ts

```diff
@@ -8496,303 +8144,6 @@ export interface ImageStoreStatic {
         ): void
 }
 
-export interface TabsReducerStatic {
-    JumpToAction(index: number): any;
-}
-
-export type TabsReducerFunction = (params: any) => any;
```

Comment:
I'm not sure where `TabsReducerStatic` came from, but it's not referenced anywhere.